apps/web/src/components/molecules/StudyDetails/StudyDetails.tsx

```diff
@@ -1,61 +1,113 @@
 import { Table } from '@nihr-ui/frontend'
 
+import type { getStudyByIdFromCPMS } from '@/lib/cpms/studies'
 import type { getStudyById } from '@/lib/studies'
 
 export interface StudyDetailsProps {
   study: NonNullable<Awaited<ReturnType<typeof getStudyById>>['data']>
+  studyInCPMS: NonNullable<Awaited<ReturnType<typeof getStudyByIdFromCPMS>>['study']>
 }
 
-export function StudyDetails({ study }: StudyDetailsProps) {
+function normalizeStudyData(study: StudyDetailsProps['study'], studyInCPMS: StudyDetailsProps['studyInCPMS']) {
+  const {
+    title: studyTitle,
+    protocolReferenceNumber: studyProtocol,
+    irasId: studyIrasId,
+    cpmsId: studyCpmsId,
+    organisationsByRole: studyOrgs,
+    managingSpeciality: studyManagingSpecialty,
+    chiefInvestigatorFirstName: studyCIFirstName,
+    chiefInvestigatorLastName: studyCILastName,
+    route: studyRoute,
+  } = study
+
+  const {
+    Title: cpmsTitle,
+    ProtocolReferenceNumber: cpmsProtocol,
+    IrasId: cpmsIrasId,
+    StudyId: cpmsStudyId,
+    organisationsByRole: cpmsOrgs,
+    ManagingSpecialty: cpmsManagingSpecialty,
+    ChiefInvestigatorFirstName: cpmsCIFirstName,
+    ChiefInvestigatorLastName: cpmsCILastName,
+    StudyRoute: cpmsRoute,
+  } = studyInCPMS
+
+  let chiefInvestigator = 'None available'
+
+  if (cpmsCIFirstName && cpmsCILastName) {
```

Comment:
Just double checking, is it correct that we resort to 'None available' even if **only** first or last name exists?

Reply:
@petecorriepa 
what would we want to see here?
the original code showed name if first name exists, otherwise shows 'None available'. see below

```
<Table.Cell>
            {study.chiefInvestigatorFirstName
              ? `${study.chiefInvestigatorFirstName} ${study.chiefInvestigatorLastName}`
              : 'None available'}
 </Table.Cell>
```

Reply:
Don't want to overcomplicate this...
We don't have any data in SE or in CPMS (where its pulled from) where the firstname is populated and the lastname isn't, or vice versa, so happy to keep it as is (as long as the name is shown if available)